src/applications/personalization/profile/components/direct-deposit/BankInfoCNPv2.jsx

```diff
@@ -0,0 +1,359 @@
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
This is a copy of the existing `BankInfoCNP` component to use on the new `DirectDepositV2` component. The only difference between this and the existing `BankInfoCNP` is its table only shows a single row containing bank info, it does not have the second row that contains "payment history" as that is it'